### ISSUE TITLE
Suggestion: Improvement studies

### ISSUE THREAD

**Issue by Kloriuwun** [binogure-studio/city-game-studio-i18n#393]
As seen in Mad Game Tycoon, that the player has the opportunity to refine or perfect a game with different studios; As for example, optimization, bug repair (not included in game developers and whether it is a studio or a different floor in the building) testers, creating real animations. For example a studio where they are used to create animations and motion capture with green background, sound studies, music etc. I know it sounds a little ambitious, but MGT did it but in a more small way. The fact that there are different studies that are aimed at refining the project would be excellent. What do you think?

**Comment by Fozehor** [comment]
`I Support It 💌`

**Comment by GeneralNW** [comment]
Agreed this would be a great addition to the gameplay but might take some time.
